tests/e2e/main_test.go

```diff
@@ -66,6 +66,11 @@ func TestMain(m *testing.M) {
 		"",
 		"DCGM-exporter image tag to use")
 
+	flag.StringVar(&suiteCfg.arguments,
+		"arguments",
+		"",
+		"DCGM-exporter command line")
+
 	flag.Parse()
```

Comment:
Arguments allow to pass a custom set of arguments to the dcgm-exporter helm chart for deployment.